packages/vuetify/src/components/VField/VField.tsx

```diff
@@ -64,7 +64,7 @@ export const makeVFieldProps = propsFactory({
   active: Boolean,
   centerAffix: {
     type: Boolean,
-    default: undefined,
+    default: false,
```

Comment:
If default is false, can just do `centerAffix: Boolean` which defaults to false.